enterprise/frontend/src/embedding-sdk/docs/README.md

```diff
@@ -0,0 +1,197 @@
+    <button
+      className="tw-text-base tw-text-yellow-900 tw-bg-slate-400 tw-rounded-lg"
+      onClick={() => {
+        alert(`Clicked ${clicked.column?.name}: ${clicked.value}`);
+        closePopover();
+      }}
+    >
+      Custom element
+    </button>
+  ),
+});
+
+const plugins = {
+  /**
+   * You will have access to default `clickActions` that Metabase render by default.
+   * So you could decide if you want to add custom actions, remove certain actions, etc.
+   */
+  mapQuestionClickActions: (clickActions, clicked) => {
+    return [
+      ...clickActions,
+      createCustomAction(clicked),
+      createCustomActionWithView(clicked),
+    ]
+  }
+}
+
+return (
+  <MetabaseProvider config={config} pluginsConfig={plugins}>
+    <InteractiveQuestion questionId={firstQuestion.id}  />
+  </MetabaseProvider>
+);
+```
+
+After you have configured Metabase embedding SDK for React, you need to make sure your `jwtProviderUri` endpoint returns JWT token that your Metabase instance can use for authentication.
+
+**Metabase embedding SDK for React only support JWT authentication.**
+
+Here is the example how you could implement an endpoint that will return JWT token that Metabase embedding SDK for react could use to authenticate your users.
+
+```ts
+import express from "express"
+import type { Request, Response } from "express"
+
+import jwt from "jsonwebtoken"
+import fetch from "node-fetch"
+
+async function metabaseAuthHandler(req: Request, res: Response) {
+  const { user } = req.session
+
+  if (!user) {
+    return res.status(401).json({
+      status: 'error',
+      message: 'not authenticated',
+    })
+  }
+
+  const token = jwt.sign(
+    {
+      email: user.email,
+      first_name: user.firstName,
+      last_name: user.lastName,
+      groups: [user.group],
+      exp: Math.round(Date.now() / 1000) + 60 * 10, // 10 minutes expiration
+    },
+    // This is the JWT signing secret in your Metabase JWT authentication setting
+    METABASE_JWT_SHARED_SECRET
+  )
+  const ssoUrl = `${METABASE_INSTANCE_URL}?token=true&jwt=${token}`
+
+  try {
+    const response = await fetch(ssoUrl, { method: 'GET' })
+    const text = await response.text()
+    const token = JSON.parse(text)
+
+    return res.status(200).json(token)
+  } catch (error) {
+    if (error instanceof Error) {
+      res.status(401).json({
+        status: 'error',
+        message: 'authentication failed',
+        error: error.message,
+      })
+    }
+  }
+}
+
+const app = express()
+
+// middleware
+app.use(
+  session({
+    secret: SESSION_SECRET,
+    resave: false,
+    saveUninitialized: true,
+    cookie: { secure: false },
+  }),
+)
+app.use(express.json())
+
+// routes
+app.get("/sso/metabase", metabaseAuthHandler)
+app.listen(PORT, () => {
+  console.log(`API running at http://localhost:${PORT}`)
+})
+```
+
+## More examples
+- [Customer Zero: Embedding SDK demo application](https://github.com/metabase/embedding-sdk-customer-zero)
```

Comment:
This is a private repo. Maybe we can remove this section for now. What do you guys think?